### DISCUSSION TITLE
Can't click on URL:s in the slider

### DISCUSSION THREAD

**Discussion by Devotee007** [nolimits4web/swiper#5428] [General]
Hi,

I have a slider with articles, that has a standard a-tag, that and the area around it is clickable. It works outside of the slider, but inside it works from time to time. Sometimes once, sometime not at all. If I refresh the page it works once. The thing that happens on clicking the slider is that it slides instead of going to the linked page. I have tried to add these:

 preventClicks: false,
preventClicksPropagation: false

But that didn't do anything. Am I missing something obvious? :)

**Comment by gavin310** [comment]
Same thing happening on certain computers in our company and not on others. A newer MacBook with the latest version of Chrome and Safari cannot click any links in the slider. An iMac one year older with the latest version of Chrome and Safari can click every link with no issues.

**Comment by linedup** [comment]
Yes I am getting exactly the same problem with a new site rollout using Swiper for the first time. Mac & Chrome.
It is very intermittent, but definitely occurs. I have never had this sort of problem previously with a slider library, so I'm keen to get to the bottom of it.

The only solution that appears to work, but is not very satisfactory is:

simulateTouch: false

This then prevents any swiping by the mouse, on desktop, which is not the end of the world - but seems odd that we have to go and disable this, in order to get the URL's working. I'm keen to hear of any alternative fix - seems like a bug to me.

**Reply by gavin310** [reply]
We couldn't figure it out since everyone is working remotely and none of the devs could duplicate the issue. Certain computers cannot interact with the slides at all, no matter what. We had to go back and remove Swiper everywhere. It's a large site with many unique sliders and tab systems that relied heavily on Swiper, so we wasted so many hours on this problem. I still have no idea what's causing it. Since some machines can use it fine and some can't use it at all, and since these computers have almost the exact same software configurations, the only thing I can think of is maybe it's the hardware acceleration that's screwing things up, since I read Swiper utilizes hardware acceleration quite a bit. Swiper seems like a great plugin, but unfortunately it's not production-ready.

**Comment by timinformatica** [comment]
I come here with the same problem and for me the solution was:

watchSlidesProgress: true
[https://github.com/nolimits4web/swiper/issues/5437#issuecomment-1122743235](url)
#5437
